Library/Homebrew/test/cli/named_args_spec.rb

```diff
@@ -179,18 +179,38 @@ def setup_unredable_cask(name)
   describe "#to_kegs" do
     before do
       (HOMEBREW_CELLAR/"foo/1.0").mkpath
+      (HOMEBREW_CELLAR/"foo/2.0").mkpath
       (HOMEBREW_CELLAR/"bar/1.0").mkpath
     end
 
     it "resolves kegs with #resolve_kegs" do
-      expect(described_class.new("foo", "bar").to_kegs.map(&:name)).to eq ["foo", "bar"]
+      expect(described_class.new("foo", "bar").to_kegs.map(&:name)).to eq ["foo", "foo", "bar"]
+    end
+
+    it "resolves kegs with multiple versions with #resolve_keg" do
+      expect(described_class.new("foo").to_kegs.map(&->(k) { k.version.version })).to eq ["1.0", "2.0"]
     end
 
     it "when there are no matching kegs returns an array of Kegs" do
       expect(described_class.new.to_kegs).to be_empty
     end
   end
 
+  describe "#to_default_kegs" do
```

Comment:
In this block can we also add an e.g. `foo/2.0` keg and test that `to_default_kegs` only returns the latest version of the keg?

Reply:
Sounds like a good idea